BEPs/BEP322.md

```diff
@@ -0,0 +1,506 @@
+  validators can easily get more extractable value from integrating with more MEV providers, delegators can earn more
+  rewards from validators.
+
+- **Improve transparency**. This specification would try to bring transparency into bsc mev market, expose profit 
+  distribution among stakeholders(validators, delegator, builders) to public.
+
+## 3. Specification
+
+### 3.1 BSC Trust Model
+
+Before we move on to the design of Builder Specification, let's have an
+analysis of the BSC trust model and compare it to Ethereum. It will
+lead to different designs for these two networks.
+
+- Validators are more trustworthy in BNB Smart Chain. In BSC, the
+  validators should delegate a lot of BNB (more than 10 thousand in
+  general) and maintain a high reputation. Misbehaviour will lead to
+  reputation damage and un-delegations. Currently, there are about
+  40 validators (more validators are coming) with 20M BNB staked, including active and inactive
+  ones. In Ethereum, the barrier to becoming a validator is very
+  low (i.e., 32 ETH), and there are more than [800 thousand Ethereum
+  validators](https://www.nasdaq.com/articles/the-most-pressing-issue-on-ethereum-is-validator-size-growth).
+  It means that a validator can be anyone, and it can enter or quit
+  easily. That's quite different from BSC.
+
+- Meanwhile, the relay role in Ethereum Builder API is supposed to be
+  trusted by both builders and validators. In BSC, the necessity of
+  introducing another trusted role is not that much. In this
+  specification, we don't utilize relay; however, it is still
+  workable to add relay between builders and validators.
+
+### 3.2 Workflow
+
+#### 3.2.1 Overall Workflow
+
+Builders and validators collaborate to propose blocks during each production interval. Two interaction options exist 
+between them: one-round and two-round. In one-round, builders include transactions with their bids, allowing validators 
+to immediately seal a block without additional transaction requests. Conversely, in two-round, builders send bids 
+without transactions initially. If selected, validators retrieve transactions through another RPC call before sealing 
+the block. The two-round process is considered more time-consuming compared to the one-round interaction. 
+
+*One-round is the default option for trusted relationship,*
+*It's highly suggest to adopt two-round interaction for builders who* 
+*1. not trust validator;* 
+*2. don't care too much about latency or adopt chance.* 
+
+**One-round(recommended and default)** interaction:
+
+<img src="./assets/bep-322/workflow_1round.png" alt="workflow" width="500"/>
+
+1) Builder submits bids with transactions to the current or multiple registered proposers.
+2) Builder gets the idea if the bid is or not successfully received from proposers' responses.
+3) Proposer picks the max valuable bid from all received bids.
```

Comment:
==> `Proposer picks the max valuable bid from all received bids at proper timing.`